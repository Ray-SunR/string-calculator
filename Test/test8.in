n a a
m b 10000 a
eb
ea
la
lb

fa
fb
s c a b
lc
wc
ea
eb
ec

n a abc
fb
s c a b
lc
wc
ea
eb
ec

fa
fb
t c a b
lc
wc
ea
eb
ec

fa
m b 0 a
lb
wb
ea
eb

n a a
m b 0 a
lb
wb
ea
eb
