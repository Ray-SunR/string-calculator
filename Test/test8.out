0
0
0

3
abc
1
b
0

0

